### PR TITLE
Updating an invitation information

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes missing information about sending invitations and prompts to users

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-my-invitations.html

### CODE REVIEW

File: src/customers/account-dashboard-my-invitations.md [hunk 1]
```diff
@@ -7,3 +7,27 @@ The My Invitations section of the customer account lists all [invitations]({% li
 
 ![]({% link images/images-ee/account-dashboard-my-invitations.png %}){: .zoom}
 _[My Invitations]({% link marketing/invitations.md %})_{:.ee-only}
+
+## Send invitation
+
+1. In the sidebar of your customer account, choose **My Invitations**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-invitations.md:13]:
```suggestion
1. In the sidebar of their account on the storefront, the customer chooses **My Invitations**.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for third-person procedure, grammar, and style

**Comment by EvheniiaKolkhiieva** [comment]
@leslietilling, thank you for reviewing!

All suggestions have been implemented, could you please approve this PR?

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@leslietilling, an error occurred during the Pull Request import.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-merchdocs-1208`.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1208`.

**Pull request merged**